### ISSUE TITLE
new resource: aws_emr_managed_scaling_policy

### ISSUE THREAD

**Issue by c4po** [terraform-providers/terraform-provider-aws#13952]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS just add the managed scaling feature for EMR cluster. This can scaling up/down instance groups and instance fleets.

### New or Affected Resource(s)

* aws_emr_managed_scaling_policy

### Potential Terraform Configuration

```hcl
resource "aws_emr_managed_scaling_policy" "samplepolicy" {
  cluster_id                      = aws_emr_cluster.sample.id
  unit_type                       = "InstanceFleetUnits"
  minimum_capacity_units          = 2
  maximum_capacity_units          = 10
  maximum_ondemand_capacity_units = 2
  maximum_core_capacity_units     = 10
}
```

### References

* https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-managed-scaling.html

**Comment by c4po** [comment]
I have the PR [13965](https://github.com/terraform-providers/terraform-provider-aws/pull/13965) for review.
But it need upgrade the aws-sdk-go to v1.32.10 for the support. current version is  v1.32.3.
Once the `go.mod` upgraded, I'll rebase my PR to make it ready.

**Comment by bflad** [comment]
Support for this new resource has been merged and will release with version 3.4.0 of the Terraform AWS Provider, later this week. Thanks to @c4po for the implementation. 👍
